### PR TITLE
Updates to sphinx

### PR DESCRIPTION
**Pull request by florentinvonhaugwitz**
# PR Description

## Please check if the PR fulfills these requirements

- [ ] Tests for the changes have been added (for bug fixes/features) (Or not needed)
- [x] Labels, reviewers and assignees have been added to the pull request
- [x] Docs have been added / updated (for bug fixes / features) (Or not needed)

## Motivation and Context

Updates to sphinx. This should remove all errors and allow each package to be installed

### CODE REVIEW

File: src/clf/create_data.py [hunk 1]
```diff
@@ -9,9 +9,10 @@
 import os
 
 import pandas as pd
-from constants import RAW_DATA_DIR, X_CLASSIFIED, X_MASKED
 from sklearn.model_selection import train_test_split
 
+from src.clf.constants import RAW_DATA_DIR, X_CLASSIFIED, X_MASKED
```
Comment (HannahPetry) [inline src/clf/create_data.py:14]: funktioniert nur mit src.clf das generieren der docs oder ist das einfach einheitlich zu anderen scripts?
Wenn ich das änder, kriege ich den Fehler "No module named 'src'"

**Review by HannahPetry — DISMISSED** [review]

File: src/clf/create_data.py (hunk 1 above)
Comment (florentinvonhaugwitz) [reply src/clf/create_data.py:14]: Du brauchst den setup.cfg und pyproject.toml file in deiner branch dafür. Und dann musst du `pip install -e .` laufen lassen
